nightfall-deployer/contracts/Config.sol

```diff
@@ -6,14 +6,17 @@ import './Ownable.sol';
 import './Structures.sol';
 
 contract Config is Ownable, Structures {
-    uint256 constant REGISTRATION_BOND = 10 wei; // TODO owner can update
+    uint256 constant MINIMUM_STAKE = 100 wei; // TODO owner can update
```

Comment:
for a future PR, these parameters need to be configurable

Reply:
Yes, defined in issue #1018